packages/input-info/src/ux-input-info.ts

```diff
@@ -2,6 +2,9 @@ import { customElement, bindable } from 'aurelia-templating';
 import { inject } from 'aurelia-dependency-injection';
 import { UxInputInfoTheme } from './ux-input-info-theme';
 import { StyleEngine, UxComponent } from '@aurelia-ux/core';
+import { UxTextArea } from '@aurelia-ux/textarea';
+import { UxInput } from '@aurelia-ux/input';
```

Comment:
@EisenbergEffect @ben-girardet theoretically, we are probably not gonna use this component without either ux-input/ux-textarea, but I think taking a direct dependency on those components are quite unnecessary. Also this forces textarea & input always used when input-info is used.
Maybe just do check against element name only?

Reply:
You mean: 

```ts
if (this.target.tagName === 'UX-INPUT' || this.target.tagName === 'UX-TEXTAREA') {
```

?? I think it's a good idea!

Reply:
@bigopon now I remember why I added the dependencies, its purely a typescript thing. If typescript doesn't know the type of `this.target` it will not accept `.maxlength` or `.value`.

<img width="846" alt="Capture d’écran 2020-01-04 à 22 01 55" src="https://user-images.githubusercontent.com/4275071/71771593-049b8400-2f3e-11ea-95eb-96f48f0741ac.png">

What is best in this situation ? Adding the dependency or using an `( ... as any)` ?

Reply:
I forgot that `.target` is the view model of either ux-input or ux-textarea.
I think in this case, we can do:
```ts
const target = this.target;
if (target.element != null) {
  const tagName = target.element.tagName;
  if (tagName === 'UX-INPUT' || tagName === 'UX-TEXTAREA') {
    ...
  }
}
```
And we just need to change the typing of target to
```ts
target: { element: Element };
```

Reply:
@bigopon https://github.com/aurelia/ux/pull/232/commits/0d2ab633905aea354ae1db0b314ef5fcb1dcf4ad